src/template/index.d.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { Resolver } from "./resolver";
+
+/**
+ * Path information of the context
+ */
+export declare interface PathInfo {
+  /**
+   * Request suffix (aka __ow_path)
+   */
+  suffix:string;
+}
+
+/**
+ * Runtime information of the context
+ */
+export declare interface RuntimeInfo {
+  /**
+   * Runtime name
+   */
+  name:string;
+
+  /**
+   * Deploy region
+   */
+  region:string;
+}
+
+/**
+ * Function information of the context
+ */
+export declare interface FunctionInfo {
+  /**
+   * Function name. Stemmed name of the function.
+   * @example 'dispatch'
+   */
+  name:string;
+
+  /**
+   * Version in dotted format.
```

Comment:
What about @ci versions?

Reply:
that's ok. I added this to highlight that it's not the the same as the `_` format we use in the AWS aliases.